spec/proformaxml/exporter_spec.rb

```diff
@@ -404,7 +404,7 @@
       end
 
       it 'add file-restrictions to submission-restrictions' do
-        expect(xml.xpath('/task/submission-restrictions/file-restriction')).to have(2).items
+        expect(xml.xpath('/task/submission-restrictions/file-restriction')).to have(3).items
```

Comment:
This change is not clear to me. Why do we expect more items here? 🤔

Reply:
I changed the factory and added one more edge-case to the list.